app/views/pages/get-started/working-with-the-code-and-files.liquid

```diff
@@ -0,0 +1,176 @@
+---
+metadata:
+  title: Working with the code and files
+  description: Follow these step-by-step tutorials to set up your development environment and deploy your first platformOS site.
+converter: markdown
+---
+
+One of the core concepts of platformOS is that you never work offline. You edit the files on your system in your favourite code editor, but you are constantly connected to your instance so that platformOS engine can interpret and parse your codebase.
+
+You write code locally, but it is executed remotely.
+
+
+## Proper platformOS directory structure
+
+In order for platformOS to understand and properly interpret your code, it needs to be organised in a specific way.
+
+The simplest method for getting the proper folder structure is to use pos-cli’s init command.
+
+Let’s start with creating an empty folder – named in this example platforms-hello – for your project and then navigating to it from the command line:
+
+<pre class="command-line" data-user="user" data-host="host"><code class="language-bash">
+cd x:\platformos-hello
+</code></pre>
+
+Now, to automatically create the proper directory structure inside that folder you just need to use:
+
+<pre class="command-line" data-user="user" data-host="host"><code class="language-bash">
+pos-cli init
+</code></pre>
+
+The command should return a success message confirming that the directory structure was created.
+
+This means that there are a bunch of new files and folders in your project’s root directory. We’ll get to what are they for in the next steps of this tutorial, or - if you want to jump in to the technical details - check out [platformOS Codebase](/developer-guide/platformos-workflow/codebase) section of this documentation.
+
+What you need to know for now is that this command creates an example HTML page with a welcome message.
+
+
+## Authenticate your environment
```

Comment:
This section needs a separate branch for users that registered using SSO, this is in progress.